static/src/js/containers/DownloadHistoryContainer/DownloadHistoryContainer.jsx

```diff
@@ -1,71 +1,87 @@
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
+import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 
-import actions from '../../actions'
 import { DownloadHistory } from '../../components/DownloadHistory/DownloadHistory'
+import RetrievalRequest from '../../util/request/retrievalRequest'
+import { addToast } from '../../util/addToast'
 
-export const mapStateToProps = (state) => ({
-  earthdataEnvironment: state.earthdataEnvironment,
-  retrievalHistory: state.retrievalHistory.history,
-  retrievalHistoryLoading: state.retrievalHistory.isLoading,
-  retrievalHistoryLoaded: state.retrievalHistory.isLoaded
-})
+export const DownloadHistoryContainer = ({
+  authToken,
+  earthdataEnvironment
+}) => {
+  const [retrievalHistory, setRetrievalHistory] = useState([])
+  const [retrievalHistoryLoading, setRetrievalHistoryLoading] = useState(false)
+  const [retrievalHistoryLoaded, setRetrievalHistoryLoaded] = useState(false)
 
-export const mapDispatchToProps = (dispatch) => ({
-  onFetchRetrievalHistory: (earthdataEnvironment) => {
-    dispatch(actions.fetchRetrievalHistory(earthdataEnvironment))
-  },
-  onDeleteRetrieval:
-    (retrievalId) => dispatch(actions.deleteRetrieval(retrievalId))
-})
+  const fetchRetrievalHistory = async () => {
+    if (!authToken) return
 
-export class DownloadHistoryContainer extends Component {
-  componentDidMount() {
-    const {
-      onFetchRetrievalHistory
-    } = this.props
+    setRetrievalHistoryLoading(true)
 
-    onFetchRetrievalHistory()
+    try {
+      const requestObject = new RetrievalRequest(authToken, earthdataEnvironment)
+      const response = await requestObject.all()
+      const { data } = response
+
+      setRetrievalHistory(data)
+      setRetrievalHistoryLoaded(true)
+    } catch (error) {
+      console.error('Error fetching retrieval history:', error)
```

Comment:
console.error in a user's browser does not help us find and fix problems, errors need to be dispatched to `handleError` like they previously were. That method handles showing the user a problem and reporting the error to a lambda where we can see the log.